bin/gce/google_cloud_mango_install.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
can you simplify into 1 install script?

Reply:
What do you mean? This script will be automatically run upon cluster creation.